docs/site/pages/dsl/dsl-schema-type.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: DSLSchema
+---
+
+## DSLSchema
+
+A `DSLSchema` type is provided in order be able to define the acceptable data types to be used and the validators to be defined for authoring the DSL data schema in our workspace. In the following example we'll generate everything we need for our DSLSchema type instance for checking based on the data types and validators available in the Player plugin `common-types-plugin`. For this purpose you can use some types and utilities found in the `@player-tools/dsl` package. 
```

Comment:
The "why"/benefits of this feature should be expanded on a bit more in this write up, especially upfront.